profile.html

```diff
@@ -37,30 +37,42 @@ <h2>Your Skills</h2>
       <button type="">Logout</button>
     </main>
 
-    <nav>
-      <ul role="list">
+    <nav class="app-grid__footer">
+      <ul role="list" class="flex">
         <li>
           <a href="./index.html"
             ><img
-              src="./Images/13439819131600005534-128.png"
+              src="./Images/home-new-48x48.svg"
               alt=""
               width="45px"
               height="45px"
```

Comment:
for images we can specify the width and height without pixels, e.g. only width="40"
the same below

Reply:
Will change it, thank you!